sdk/containerregistry/azure-containerregistry/setup.py

```diff
@@ -45,6 +45,7 @@
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
```

Comment:
You can remove 3.7 & 3.8.

Reply:
Why 3.8?

Reply:
Sorry. Removing 3.7. :)

I wanted to say python_requires=">=3.7", to python_requires=">=3.8",

But let's double check with Scott if CI is ready.

Reply:
@scbedd

Reply:
I find tests on windows2022 is using python3.7, does that mean it's not good to remove 3.7 for now?
![image](https://github.com/Azure/azure-sdk-for-python/assets/20915761/bdd90321-5b87-4634-a89b-dd07a94a20d8)

Reply:
IMO you can remove the `python 3.7` claim now. I'm intending on getting `3.7` pulled out of the matrix very soon, but I'm catching up after vacay and just haven't gotten to it yet.

Reply:
I've removed the `python 3.7` but the pipeline on `Test windowns2022_37` is still there and failing. @scbedd Do you know how to get CI pass?

Reply:
We are working on it. It will be clean when the PR is merged.

https://github.com/Azure/azure-sdk-for-python/pull/33722